src/publish/confluence/api/index.ts

```diff
@@ -94,7 +94,7 @@ export class ConfluenceClient {
     const CQL_CONTEXT =
       "%7B%22contentStatuses%22%3A%5B%22archived%22%2C%20%22current%22%2C%20%22draft%22%5D%7D"; //{"contentStatuses":["archived", "current", "draft"]}
 
-    cql = `${cql}&spaces=${space.key}&cqlcontext=${CQL_CONTEXT}`;
+    cql = `${cql}%20and%20space=${space.key}&cqlcontext=${CQL_CONTEXT}`;
```

Comment:
It seems indeed this is not good : https://developer.atlassian.com/cloud/confluence/cql-keywords/

Does this means the other `&` in `&cqlcontext=${CQL_CONTEXT}` is also wrong ?

Reply:
I believe the `cqlcontext` is fine, because it is its own query parameter. The doc I looked at was here:

https://developer.atlassian.com/cloud/confluence/rest/v1/api-group-search/#api-wiki-rest-api-search-get-request

I can see both `cql` and `cqlcontext` are query parameters, so they can be separated with `&`.